6-AdvancedScenarios/4-hybrid-SPA/README.md

```diff
@@ -0,0 +1,412 @@
+
+## Setup the sample
+
+### Step 1: Clone or download this repository
+
+From your shell or command line:
+
+```console
+    git clone https://github.com/Azure-Samples/ms-identity-javascript-react-tutorial.git
+```
+
+or download and extract the repository .zip file.
+
+> :warning: To avoid path length limitations on Windows, we recommend cloning into a directory near the root of your drive.
+
+### Step 2: Install project dependencies
+
+* Setup the service app:
+
+    ```console
+        cd 6-AdvancedScenarios\4-hybrid-SPA\App
+        npm install
+    ```
+
+* Setup the client app:
+
+    ```console
+        cd cd 6-AdvancedScenarios\4-hybrid-SPA\App\client
+        npm install
+    ```
+
+### Step 3: Application Registration
+
+There is one project in this sample. To register it, you can:
+
+* follow the steps below for manually register your apps
+* or use PowerShell scripts that:
+  * **automatically** creates the Azure AD applications and related objects (passwords, permissions, dependencies) for you.
+  * modify the projects' configuration files.
+
+<details>
+   <summary>Expand this section if you want to use this automation:</summary>
+
+  > **WARNING**: If you have never used **Azure AD Powershell** before, we recommend you go through the [App Creation Scripts guide](./AppCreationScripts/AppCreationScripts.md) once to ensure that your environment is prepared correctly for this step.
+
+  1. On Windows, run PowerShell as **Administrator** and navigate to the root of the cloned directory
+
+  1. In PowerShell run:
+
+```PowerShell
+    Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope Process -Force
+    ```
+
+  1. Run the script to create your Azure AD application and configure the code of the sample application accordingly.
+  1. For interactive process - in PowerShell run:
+
+    ```PowerShell
+      cd .\AppCreationScripts\
+      .\Configure.ps1 -TenantId "[Optional] - your tenant id" -Environment "[Optional] - Azure environment, defaults to 'Global'"
+     ```
+
+  > Other ways of running the scripts are described in [App Creation Scripts guide](./AppCreationScripts/AppCreationScripts.md)
+  > The scripts also provide a guide to automated application registration, configuration and removal which can help in your CI/CD scenarios.
+
+ </details>
+
+### Choose the Azure AD tenant where you want to create your applications
+
+1. Sign in to the [Azure portal](https://portal.azure.com).
+1. If your account is present in more than one Azure AD tenant, select your profile at the top right corner in the menu on top of the page, and then **switch directory** to change your portal session to the desired Azure AD tenant.
+
+### Register the service app (msal-hybrid-spa)
```

Comment:
>msal-hybrid-spa [](http://example.com/codeflow?start=30&length=15)

rename to msal-nodejs-webapp ?

Reply:
If we rename the sample to msal-nodejs-webapp shouldn't the sample live in the node chapters rather than React?